docs/configuration.md

```diff
@@ -532,6 +532,8 @@ It uses configuration from `exchange.pair_whitelist` and `exchange.pair_blacklis
 
 `refresh_period` allows setting the period (in seconds), at which the pairlist will be refreshed. Defaults to 1800s (30 minutes).
 
+`VolumePairList` is based on the volume of the last 24 hours.
```

Comment:
This is true only for quoteVolume, proposed change instead:

```
`VolumePairList` is based on the ticker data, as reported by the ccxt library:

* The `bidVolume` is the volume (amount) of current best bid in the orderbook.
* The `askVolume` is the volume (amount) of current best ask in the orderbook.
* The `quoteVolume` is the amount of quote (stake) currency traded (bought or sold) in last 24 hours.
```

Reply:
Interesting, here the best bid and best ask is an aggregated level of the order book? Because if it just the volume of best bid or ask I not sure how useful it can be to filter markets?

Reply:
all this actually depends on market. Even quoteVolume (can be other value, than 24-h rolling aggregate)

Reply:
Press "Commit suggestion" if you agree with this my proposed change.

Reply:
I don't have the button commit suggestion I don't know why. For bidvolume and askvolume I don't understand how it can be useful to filter markets by the volume of the best bid or ask. If btc has best bid of 30 dollar amount anf shit coin 2000 dollar bid it will remove btc and keep shit coin? Maybe I missed smt? Tks

Reply:
me too, I also do not know how other options (bidVolume, askVolume) can be used in practice, but it's legacy. existing values for this setting, Your description is wrong since it's only applicable to one of the available options.

Reply:
commited by myself, pls review

Reply:
I know my description is wrong :-) Just trying to understand.

Reply:
@gaugau3000 can I merge this PR with my changes?

we often use a short mark "LGTM 👍" for this (due to historical reasons), just to log in the comments explicit agreement with changes made, you may use any words, but I do not see your explicit agreement with the additional changes above... (just to be it explicit)

Reply:
LGTM :+1: you can merge. By the way you are right the discussion about bidvolume and askVolume should be in a topic and far from this pull request (like the S of SOLID in programming) .